aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -66,87 +66,52 @@ Recommendations for global deployments to geographical data centers:
 
-* Configure the option in `appsettings.json`:
+:::moniker range=">= aspnetcore-8.0"
 
-  ```json
-  "Azure:SignalR:ServerStickyMode": "Required"
-  ```
+<!-- UPDATE 9.0 Update section to only cross-link stateful
+                reconnect guidance after the feature is
+                supported with Azure SignalR Service. -->
 
 > [!NOTE]
-> The following error is thrown by an app that hasn't enabled session affinity:
->
-> > :::no-loc text="blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
-
-To provision the Azure SignalR Service for an app in Visual Studio:
-
-1. Create an Azure Apps publish profile in Visual Studio for the app.
-1. Add the **Azure SignalR Service** dependency to the profile. If the Azure subscription doesn't have a pre-existing Azure SignalR Service instance to assign to the app, select **Create a new Azure SignalR Service instance** to provision a new service instance.
-1. Publish the app to Azure.
-
-Provisioning the Azure SignalR Service in Visual Studio automatically adds the SignalR connection string to the app service's configuration.
-
-## Azure App Service
-
-:::moniker range=">= aspnetcore-8.0"
-
-Hosting a Blazor Web App that uses interactive server-side rendering on Azure App Service requires configuration for Application Request Routing (ARR) affinity and WebSockets. The App Service should also be appropriately globally distributed to reduce UI latency. Using the Azure SignalR Service when hosting on Azure App Service isn't required.
+> [Stateful reconnect](xref:signalr/configuration#configure-stateful-reconnect) (<xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilderHttpExtensions.WithStatefulReconnect%2A>) was released with .NET 8 but isn't currently supported for the Azure SignalR Service. For more information, see [Stateful Reconnect Support? (`Azure/azure-signalr` #1878)](https://github.com/Azure/azure-signalr/issues/1878).
 
 :::moniker-end
 
-:::moniker range="< aspnetcore-8.0"
+For guidance on deployment and configuration for the Azure SignalR Service:
 
-Hosting a Blazor Server app on Azure App Service requires configuration for Application Request Routing (ARR) affinity and WebSockets. The App Service should also be appropriately globally distributed to reduce UI latency. Using the Azure SignalR Service when hosting on Azure App Service isn't required.
+* Only when the app is in production on Azure App Service, see <xref:signalr/publish-to-azure-web-app>.
+* When you want the app to adopt the Azure SignalR Service during development, see [Build Blazor Server chat app: Further topic: Enable Azure SignalR Service in local development](https://github.com/aspnet/AzureSignalR-samples/tree/main/samples/BlazorChat#further-topic-enable-azure-signalr-service-in-local-development).
```

Comment:
Again, if your Blazor app is hosted on Azure App Service, then I don't believe the Azure SignalR Service is needed.

Reply:
Confused ... that isn't what this states. This is in the section on the Azure SignalR Service and states that ...

* If you want to use it in production, see the cross-linked guidance (Line 99).
* If you want to use it locally, see the cross-linked guidance (Line 100).

This doesn't make any statements about it being required. An earlier line even says ...

> If your hosting environment already handles these concerns, using the Azure SignalR Service isn't necessary.

Reply:
I'm saying that there is really never a need to use the Azure SignalR Service if your Blazor app is hosted on Azure App Service. So, the first bullet on line 99 is providing guidance for a non-scenario for Blazor apps. I also don't know of a scenario for using the Azure SignalR Service with Blazor for local development.

Reply:
Ok ... gotcha. According to the chat sample, it's a thing ...

https://github.com/aspnet/AzureSignalR-samples/tree/main/samples/BlazorChat#further-topic-enable-azure-signalr-service-in-local-development

... and my understanding is that if the app is deployed with the service (profile publish out of VS) ***the service isn't local***. The service works on the Azure App instance(s) after deployment. None of the items they cover in that section ☝️ need to be done with the app, and none of the app's settings need to be adjusted in the app or in Azure. That's what I'm gleaning from their coverage.